app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
Maybe you can use `I18n` for the texts in the emails. You would have the text in `config/locales/en.yml`, and use them in the email views like `I18n.t('key_in_en.yml')`.
For example for this line you can have in the `en.yml`:
```
mailer:
  confirmation:
    title: "Welcome %{email}!"
```

And you can use it in the view like:
```
<%= t('mailer.confirmation.title', email: @email) %>
```